### PR TITLE
Display better error for invalid groups

### PR DESCRIPTION
**Pull request by bkutil**
When bulk editing nodes, currently the default error message is
displayed, when the group contains invalid characters.

 'Failed to save the nodes X.'

This patch adds correct handling of such case and the UI will
display a more informative message:

 'The group should start ...'

cc @tboerger @vuntz

### CODE REVIEW

File: crowbar_framework/app/controllers/nodes_controller.rb [hunk 1]
```diff
@@ -178,6 +180,8 @@ def bulk
         "nodes.list.duplicate_alias"
       when @report[:duplicate_publics]
         "nodes.list.duplicate_public"
+      when @report[:group_error]
+        "nodes.list.group_error"
```
Comment (vuntz) [inline crowbar_framework/app/controllers/nodes_controller.rb]: This won't work as this string requires passing `:node` in `I18n.t`.

**Comment by tboerger** [comment]
Overall this code really sucks. But i'm fine with this fix of error message for now.

File: crowbar_framework/app/controllers/nodes_controller.rb (hunk 1 above)
Comment (bkutil) [inline crowbar_framework/app/controllers/nodes_controller.rb]: Fixed

**Comment by vuntz** [comment]
Not tested, but +1 assuming someone tested it :-)

**Comment by bkutil** [comment]
> 10:03 <cmueller> patch tested. works great

**Pull request merged**